servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpLoadBalancerFactory.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
```

Comment:
We should preserve the original year in the copyright header and add a new year: 2020-2021, 2024